packages/react-components/src/organisms/ComplianceTable.tsx

```diff
@@ -1,67 +1,65 @@
 import {
   ComplianceSortingDirection,
+  ManuscriptPutRequest,
+  ManuscriptResponse,
+  ManuscriptStatus,
   PartialManuscriptResponse,
   SortCompliance,
 } from '@asap-hub/model';
 import { css } from '@emotion/react';
-import { ComponentProps } from 'react';
-import { Card } from '../atoms';
-import { borderRadius } from '../card';
-import { charcoal, neutral200, steel } from '../colors';
-import { rem, tabletScreen } from '../pixels';
+import { ComponentProps, useState } from 'react';
+import { Card, Pill } from '../atoms';
+import { charcoal, steel } from '../colors';
+import { rem } from '../pixels';
+import ComplianceAssignUsersModal, {
+  AssignedUsersFormData,
+} from './ComplianceAssignUsersModal';
 import ComplianceTableRow from './ComplianceTableRow';
+import ConfirmStatusChangeModal from './ConfirmStatusChangeModal';
 
 const container = css({
-  display: 'grid',
-  paddingTop: rem(32),
-});
-
-const gridTitleStyles = css({
-  display: 'none',
-  [`@media (min-width: ${tabletScreen.min}px)`]: {
-    display: 'inherit',
-    paddingBottom: rem(16),
-  },
-});
-
-const rowStyles = css({
-  display: 'grid',
-  padding: `${rem(20)} ${rem(24)} 0`,
-  borderBottom: `1px solid ${steel.rgb}`,
-  ':first-of-type': {
-    borderBottom: 'none',
+  paddingTop: rem(8),
+  overflowX: 'auto',
+  table: {
+    borderSpacing: 0,
+    width: 'max-content',
```

Comment:
I think we could have something like
```suggestion
    width: '100%',
    minWidth: 'max-content',
```

so in large screens the width is expanded. It looks like this in my big monitor
<img width="2560" src="https://github.com/user-attachments/assets/b7435621-29bc-477d-962e-381539838877" />

To see how it looks like, it's always possible to use this "device toolbar"
<img width="2560" src="https://github.com/user-attachments/assets/d92ecd11-0c16-4736-bc21-f1919223d573" />

my monitor for example is 2560 x 1440

Reply:
Thanks!